aidial_adapter_openai/routers/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .chat_completion import chat_completion
```

Comment:
This folder should be rather called `routes` or `endpoints` instread of `routers`.
A router is a component which groups similar endpoints _(e.g. the one starting with the same prefix)_: https://fastapi.tiangolo.com/tutorial/bigger-applications/#an-example-file-structure

Here you are defining handlers for individual endpoints.

Reply:
fixed, renamed to `endpoints`